episodes/chapter6.md

```diff
@@ -0,0 +1,570 @@
+
+
+### Where is the node configuration file?
+
+You can always use the `v6 node files` command to check the location of the node configuration file,
+
+```bash
+v6 node files
+```
+
+It will ask you which node you want to see. You can choose the one you just created,
+
+```bash
+? Select the configuration you want to use: (Use arrow keys)
+ » node1
+   node2
+   node3
+```
+
+then, it will print out the path of the configuration file,
+
+```bash
+? Select the configuration you want to use: node1
+2024-05-24 15:52:13 - context        - INFO     - ---------------------------------------------
+2024-05-24 15:52:13 - context        - INFO     -  Welcome to
+2024-05-24 15:52:13 - context        - INFO     -                   _                     __
+2024-05-24 15:52:13 - context        - INFO     -                  | |                   / /
+2024-05-24 15:52:13 - context        - INFO     - __   ____ _ _ __ | |_ __ _  __ _  ___ / /_
+2024-05-24 15:52:13 - context        - INFO     - \ \ / / _` | '_ \| __/ _` |/ _` |/ _ \ '_ \
+2024-05-24 15:52:13 - context        - INFO     -  \ V / (_| | | | | || (_| | (_| |  __/ (_) |
+2024-05-24 15:52:13 - context        - INFO     -   \_/ \__,_|_| |_|\__\__,_|\__, |\___|\___/
+2024-05-24 15:52:13 - context        - INFO     -                             __/ |
+2024-05-24 15:52:13 - context        - INFO     -                            |___/
+2024-05-24 15:52:13 - context        - INFO     -
+2024-05-24 15:52:13 - context        - INFO     -  --> Join us on Discord! https://discord.gg/rwRvwyK
+2024-05-24 15:52:13 - context        - INFO     -  --> Docs: https://docs.vantage6.ai
+2024-05-24 15:52:13 - context        - INFO     -  --> Blog: https://vantage6.ai
+2024-05-24 15:52:13 - context        - INFO     - ------------------------------------------------------------
+2024-05-24 15:52:13 - context        - INFO     - Cite us!
+2024-05-24 15:52:13 - context        - INFO     - If you publish your findings obtained using vantage6,
+2024-05-24 15:52:13 - context        - INFO     - please cite the proper sources as mentioned in:
+2024-05-24 15:52:13 - context        - INFO     - https://vantage6.ai/vantage6/references
+2024-05-24 15:52:13 - context        - INFO     - ------------------------------------------------------------
+2024-05-24 15:52:13 - context        - INFO     - Started application vantage6
+2024-05-24 15:52:13 - context        - INFO     - Current working directory is '/Users/clgeng/github/advantage/vantage6-workshop'
+2024-05-24 15:52:13 - context        - INFO     - Successfully loaded configuration from '/Users/clgeng/Library/Application Support/vantage6/node/node1.yaml'
+2024-05-24 15:52:13 - context        - INFO     - Logging to '/Users/clgeng/Library/Logs/vantage6/node/node1/node_user.log'
+2024-05-24 15:52:13 - context        - INFO     - Common package version '4.1.0'
+2024-05-24 15:52:13 - context        - INFO     - vantage6 version '4.1.0'
```

Comment:
Maybe we don't have to print this message above? 
(I actually think that we should remove it at some point for this command, but that has low prio)

Reply:
Here I have some doubts. I agree that the message is quite long, but I also think that it is nice to show somewhere what the output should look like. Maybe again as part of the solution to the challenge?